composer.json

```
@@ -19,7 +19,7 @@
         "ext-json": "*",
         "psr/http-message" : "^1.0",
         "pimple/pimple"    : "~3.0",
-        "guzzlehttp/guzzle": "^6.2"
+        "guzzlehttp/guzzle": "7.2"
```

```suggestion
        "guzzlehttp/guzzle": ^6.2|^7.2"
```

might resolve both issues